### DISCUSSION TITLE
starting cortx-hare with dual nodes

### DISCUSSION THREAD

**Discussion by kevinxyliu** [Seagate/cortx#1500] [General]
Hi,

I am trying to start cortx-hare with dual nodes by following the Hare User Guide, and I have access to two nodes that can ssh into each other using IP addresses. According to the guide, prior to running "hctl bootstrap" I need to configure a CDF that specifies parameters such as "hostname" and "data_iface" (I think the sample CDF that I need to use is [here](https://github.com/Seagate/cortx-hare/blob/main/cfgen/examples/ldr1-cluster.yaml)) However, what should I set as the hostname for the two nodes in the CDF? I think I can set one of them as localhost, but for the other one, is it required for me to use a particular name? I tried using random names for the other hostname, but kept getting errors when running "hctl bootstrap" that the hostname is not recognized. Thanks a lot!

**Comment by huanghua78** [comment]
hostname, is the host name that can be used to resolve to IP address. It is not a random name for a node.
Please see gethostbyname(3) and hostname(7).

**Reply by kevinxyliu** [reply]
Thanks for the response. I tried running gethostbyname(3) but it seems that the hostname returned by it still does not work. Just to clarify, is it possible to run dual-node setup if I have two nodes that can ssh into each other (without any further connections)? If I need to run the dual-node setup on one particular node, how shall I configure the information in the other node with the IP address? I am currently setting the IP address using the "data_iface_ip_addr" field, but the hostname returned by gethostname(3) on the other node is still not recognized when running "hctl bootstrap". Thanks!

**Reply by hessio** [reply]
Hi @kevinxyliu yes you should be able to create this setup once they are able to connect via ssh. You will need to build and install Motr code on both nodes but HARE will only need to be installed and built on one of the nodes.
